book/chapters/jets.ipynb

```diff
@@ -357,100 +343,75 @@
+    "More complex objects can come back as well. For example, vectors of floats (`std::vector<float>`). Here is a sample with the sum track $p_T$ when track $p_T > 500$ MeV.\n",
+    "\n",
+    "```{warning}\n",
+    "This example has not been updated to r22 and ServiceX 3 yet.\n",
+    "```"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 31,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "application/vnd.jupyter.widget-view+json": {
-       "model_id": "b925344b584b4f0ea7766618e47a7ef0",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "rucio://mc16_13TeV:m...:   0%|          | 0/9000000000.0 [00:00]"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    },
-    {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "e269eae434684aae8158e5128aaf6605",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "        rucio://mc16_13TeV:m... Downloaded:   0%|          | 0/9000000000.0 [00:00]"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    },
-    {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "66ee92305c62400686fb178f2481bce8",
+       "model_id": "302be45b599b4bcb90247161a9d947ed",
        "version_major": 2,
        "version_minor": 0
       },
       "text/plain": [
-       "rucio://mc16_13TeV:m...:   0%|          | 0/9000000000.0 [00:00]"
+       "Output()"
       ]
      },
      "metadata": {},
      "output_type": "display_data"
     },
     {
      "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "537e4f27896c4b209ca738139554d147",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "        rucio://mc16_13TeV:m... Downloaded:   0%|          | 0/9000000000.0 [00:00]"
-      ]
+      "text/html": [
+       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"></pre>\n"
+      ],
+      "text/plain": []
      },
      "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
-    "sum_pt = (ds\n",
-    "          .SelectMany(lambda e: e.Jets(\"AntiKt4EMPFlowJets\"))\n",
-    "          .Where(lambda j: (j.pt() / 1000) > 31)\n",
-    "          .SelectMany(lambda j: j.getAttribute[cpp_vfloat]('SumPtTrkPt500'))\n",
-    "          .AsAwkwardArray('sum_pt')\n",
-    "          .value())"
+    "query = FuncADLQueryPHYSLITE()\n",
+    "jets_per_event = query.Select(lambda e: e.Jets(\"AnalysisJets\"))\n",
+    "jet_info_per_event = jets_per_event.Select(\n",
+    "    lambda jets: {\n",
+    "        'sum_pt': jets.Select(lambda j: j.getAttribute[cpp_vfloat]('SumPtTrkPt500'))\n",
+    "    }\n",
+    ")\n",
+    "\n",
+    "files = deliver_files(jet_info_per_event,sx_f)"
```

Comment:
Yeah - having trouble reading this. Isn't the old code more readable as to what is going on?